packages/react/src/SubNav/SubNav.tsx

```diff
@@ -413,7 +413,7 @@ const SubNavLinkWithSubmenu = forwardRef<HTMLDivElement, SubNavLinkProps>(
           </button>
         )}
 
-        <div id={submenuId} className={styles['SubNav__sub-menu-children']} aria-hidden={!isExpanded}>
+        <div id={submenuId} className={styles['SubNav__sub-menu-children']} aria-hidden={isLarge && !isExpanded}>
```

Comment:
Could we add some test coverage for this please to prevent future regression and awareness? Can be added in follow-up to avoid delaying this.

Reply:
I'll jump on that later today